legate/legate_kvikio/tile.py

```diff
@@ -102,3 +102,66 @@ def read_tiles(
         The start coordinate of the tiles
     """
     return _tile_read_write(TaskOpCode.TILE_READ, ary, dirpath, tile_shape, tile_start)
+
+
+def read_tiles_by_offsets(
+    ary: cunumeric.ndarray,
+    filepath: Iterable[pathlib.Path | str],
+    offsets: Tuple[int],
+    sizes: Tuple[int],
+    tile_shape: Tuple[int],
+) -> None:
```

Comment:
FWIW, even though this modifies in place, I think for API uniformity one could return an `ndarray` (just returning `ary` at the end of the function). WDYT?

Reply:
OK, thanks.